src/mount.fuse.ceph

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
i cannot find the python3 dependency in `debian/control` for ceph-fuse . or in `ceph.spec.in` . `ceph-fuse` claims to depend on `fuse` now.

Reply:
So we also need to add the python3 dependency for ceph-fuse? @badone, can you make that change?